src/api/user-recommendations/controllers/user-recommendations.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const NodeCache = require('node-cache');
+
+/**
+ * Read the documentation (https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#core-controllers)
+ * to customize this controller
+ */
+
+const myCache = new NodeCache({ stdTTL: 60 });
+
+module.exports = {
+  async recommend(ctx) {
+    let users = myCache.get('randomUsers');
```

Comment:
So this cache will only have 1 key?

Reply:
Yes, having all of the random users under one key makes it easy to access all of them at the same time. Alternatively I could have a key for each random user if that would better.

Reply:
I think that is a product question. I'm okay with reusing the same users if the request comes within the same minute (TTL of cache).